kotlin-react/src/main/kotlin/react/ComponentClass.kt

```diff
@@ -7,11 +7,22 @@ external interface ComponentClass<in P : RProps> :
     ComponentType<P>,
     RComponentClassStatics<RProps, State, Context<*>?>
 
+@Deprecated(
+    message = "Legacy type alias",
+    replaceWith = ReplaceWith("ComponentClass", "react.ComponentClass"),
+)
 typealias RClass<P> = ComponentClass<P>
 
-val <P : RProps> KClass<out Component<P, *>>.rClass: ComponentClass<P>
+val <P : RProps> KClass<out Component<P, *>>.react: ComponentClass<P>
     get() = js.unsafeCast<ComponentClass<P>>()
 
+@Deprecated(
+    message = "Legacy type alias",
+    replaceWith = ReplaceWith(".react"),
```

Comment:
I don't know how to config `ReplaceWith` correctly for extensions.
It would be great if anyone can help me